src/sentry/static/sentry/app/utils/performance/quickTrace/traceFullQuery.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import GenericDiscoverQuery from 'app/utils/discover/genericDiscoverQuery';
+import {
+  TraceFull,
+  TraceFullQueryChildrenProps,
+  TraceProps,
+  TraceRequestProps,
+} from 'app/utils/performance/quickTrace/types';
+import {
+  beforeFetch,
+  getQuickTraceRequestPayload,
+  isTransaction,
+  makeEventView,
+} from 'app/utils/performance/quickTrace/utils';
+import withApi from 'app/utils/withApi';
+
+type QueryProps = Omit<TraceRequestProps, 'eventView'> & {
+  children: (props: TraceFullQueryChildrenProps) => React.ReactNode;
+};
+
+function EmptyTrace({children}: Pick<QueryProps, 'children'>) {
+  return (
+    <React.Fragment>
+      {children({
+        isLoading: false,
+        error: null,
+        trace: null,
+        type: 'full',
```

Comment:
Should this be empty trace type?

```suggestion
        type: 'empty',
```